pkg/v1/tkg/tkgctl/get_cluster.go

```diff
@@ -57,3 +57,19 @@ func (t *tkgctl) GetClusters(options ListTKGClustersOptions) ([]client.ClusterIn
 
 	return clusters, nil
 }
+
+func (t *tkgctl) IsClusterExists(clustername, namespace string) (bool, error) {
+	clusters, err := t.GetClusters(ListTKGClustersOptions{
+		Namespace: namespace,
+		IncludeMC: true,
+	})
+	if err != nil {
+		return false, err
+	}
+	for _, cluster := range clusters {
+		if cluster.Name == clustername && cluster.Namespace == namespace {
```

Comment:
Just noticed that we have updated condition to also check the `namespace` as well. Previously we were just checking the cluster name irrespective of the namespace.

Reply:
okay, thanks.